docs/intro.rst

```diff
@@ -304,9 +304,32 @@ ToFloat
 .......
 
 
-ToDecemal
+ToDecimal
 .........
+Checks that value is a Decimal.
+If value is a string converts this string to Decimal::
```

Comment:
I'm not sure that this point is needed here.
If u think that it is so we need to add this description to each checker. 

Also, value can be not only string. It can be an integer or float.

Reply:
I think we need short point/description of what does the checker do. Going to add similar to other checkers. Going to extend description to string with int and float, thanks.

Anyway, does it make sense, or you probably have more precise description for similar checkers?